docs/admin/backup.rst

```diff
@@ -68,8 +68,8 @@ Using custom backup storage
 ---------------------------
 
 You can also use your own storage for the backups. SSH can be used to store
-backups on the remote destination if the target server has `BorgBackup`_
-installed.
+backups in the remote destination, the target server needs to have 
```

Comment:
`on` probably was correct here...

Reply:
I should search while I was doing the review. I did now so I am sure that `at` is the best-choice preposition here. :) 

"In the destination" is OK, too.

Reply:
ok, let it be than...